src/api/cpp/cvc5.cpp

```diff
@@ -6656,7 +6664,9 @@ Result Solver::checkEntailed(const std::vector<Term>& terms) const
          "(try --incremental)";
   CVC5_API_SOLVER_CHECK_TERMS(terms);
   //////// all checks before this line
-  return d_smtEngine->checkEntailed(Term::termVectorToNodes(terms));
+  cvc5::Result r = d_smtEngine->checkEntailed(Term::termVectorToNodes(terms));
+  d_smtEngine->printStatisticsDiff();
```

Comment:
Can we actually push this further into SmtEngine::checkEntailed / SmtEngine::checkSat?

Reply:
I've put it into `checkSatInternal`, so it only lives in a single place. (well, actually two when an exception is thrown...)